### ISSUE TITLE
question about lip weights in loss function

### ISSUE THREAD

**Issue by 7aughing** [ElvishElvis/68-Retinaface-Pytorch-version#10]
hi. 
Thanks for sharing such a nice code.
When I tried your code, I found in losses.py, there are two lines to add lip weighted when compute landmarks loss, but seems idx 99 and idx 37 not the lips idx. 
![image](https://user-images.githubusercontent.com/30593398/114291801-c84dc080-9abc-11eb-8138-d2e052be6c6f.png)
May I ask which two points 99 and 37 represent?
